docs/server/upgrading.rst

```diff
@@ -3,170 +3,259 @@
-     uses them.
+.. _upgrading#check-settings:
 
-* Perform the database schema and data upgrade by running:
+Update and check your settings
+------------------------------
 
-  .. code-block:: bash
+You should now update your custom Pootle settings to add, remove or adjust any
+settings that have changed. You may want to view the latest 
+:ref:`available settings <settings#available>`.
 
-     (env) $ pootle migrate
+You can check to see if there are any issues with your configuration
+settings that need to be resolved:
 
+.. code-block:: bash
 
-* Reapply your custom changes to Pootle code, templates or styling. Read about
-  :doc:`customization of style sheets and templates
-  </developers/customization>` to adjust your customizations to the correct
-  locations and approach in the new release.
+   (env) $ pootle check
 
-  .. note:: If you have customized the CSS styling or the JavaScript code you
-     will have to run the following commands to update the static assets:
+.. note:: If you are upgrading from a version of Pootle that uses
+   :file:`localsettings.py` then you may want to merge your old custom settings
+   with your :ref:`settings conf file <settings#customizing>` (default location
+   :file:`~/.pootle/pootle.conf`).
 
-     .. code-block:: bash
 
-       (env) $ cd $pootle_dir/pootle/static/js/
-       (env) $ npm install
-       (env) $ npm update
-       (env) $ pootle webpack
-       (env) $ pootle collectstatic --noinput --clear -i node_modules -i *.jsx
-       (env) $ pootle assets build
+.. _upgrading#start-rq:
 
+Start an RQ Worker
+------------------
 
-     ``$pootle_dir`` is the directory where :command:`pip` installed Pootle. Its
-     location depends on your settings for :command:`pip`, but by default it
-     should be :file:`~/.virtualenvs/env/lib/python2.7/site-packages/`.
+Statistics tracking and various other background processes are managed by `RQ
+<http://python-rq.org/>`_.  The :djadmin:`rqworker` command needs to be run
+continuously in order to process the jobs.
 
+If you have not already done so you should
+:ref:`install and start a Redis server <requirements#packages>`.
 
-* Finally, restart your server.
+You can start the worker in the background with the following command:
 
+.. code-block:: bash
 
-.. _upgrading#post-upgrade:
+   (env) $ pootle rqworker &
 
-Post-upgrade adjustments
-------------------------
+In a production environment you may want to :ref:`run RQ workers as services
+<pootle#running_as_a_service>`.
+
+See here for :ref:`further information about RQ jobs in Pootle <rq>`.
+
+
+.. _upgrading#schema-migration:
+
+Migrate your database schema
+----------------------------
+
+Once you have updated your settings you can perform the database schema and
+data upgrade by running. This needs to be done in a few steps:
+
+.. code-block:: bash
+
+   (env) $ pootle migrate accounts 0002 --fake
+   (env) $ pootle migrate pootle_translationproject 0002 --fake
+   (env) $ pootle migrate
```

Comment:
Lets keep this for now. If in the future this becomes an increasing list of migrate commands then we will reconsider.